cypress/cypress/integration/login.spec.ts

```diff
@@ -45,23 +44,22 @@ describe("Login", () => {
 
   it("enabled if GOOGLE_CLIENT_ID is set", () => {
     cySetup(cy);
-    cyInterceptGraphQL(cy, [cyMockLessons()]);
-    cy.intercept("**/config", { GOOGLE_CLIENT_ID: "test" });
+    cyInterceptGraphQL(cy, [cyMockLessons(), cyMockConfig()]);
     cy.visit("/");
     cy.get("[data-cy=login]").should("not.be.disabled");
   });
 
   it("shows logout if logged in", () => {
     cySetup(cy);
-    cyInterceptGraphQL(cy, [cyLogin(cy), cyMockLessons()]);
+    cyInterceptGraphQL(cy, [cyLogin(cy), cyMockLessons(), cyMockConfig()]);
     cy.visit("/");
     cy.get("[data-cy=logout]").should("not.be.disabled");
     cy.get("[data-cy=logout]").contains("Kayla");
   });
 
   it("logs out", () => {
     cySetup(cy);
-    cyInterceptGraphQL(cy, [cyLogin(cy), cyMockLessons()]);
+    cyInterceptGraphQL(cy, [cyLogin(cy), cyMockLessons(), cyMockConfig()]);
```

Comment:
since almost every test has the same boilerplate:

```typescript
cyInterceptGraphQL(cy, [cyLogin(cy), cyMockLessons(), cyMockConfig()]);
```

...can we have a helper function like 

```typescript
cyMockDefault(cy)
```

...that makes those 3 calls and use it everywhere we do the same 3 calls?